Lib/idlelib/debugger.py

```diff
@@ -130,80 +154,187 @@ def make_gui(self):
         pyshell = self.pyshell
         self.flist = pyshell.flist
         self.root = root = pyshell.root
+        self.clickable_cursor = 'hand2'
+        windowingsystem = self.root.tk.call('tk', 'windowingsystem')
+        if windowingsystem == 'aqua':
```

Comment:
I used @E-Paine's suggestion... @terryjreedy I agree it would be very convenient if tkinter more visibly exposed this; also agree about a central place for what are effectively application-level globals

Reply:
Resolved